config/routes.rb

```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Defines the root path route ("/")
+  
+  root "aux#splash"
+  devise_for :users
+  resources :groups do
+    resources :entities
+  end
```

Comment:
* Kindly ensure that only the necessary routes are made accessible this is to prevent compromise of your app by users with bad intentions, you can use the `only` keyword to make open only the routes you need for example `resources :products, only: [:index, :show]` lets say the products component has only index and show actions you should write your routes this way so that users can only access those two routes only